index.js

```diff
@@ -22,13 +22,17 @@ module.exports = function(date) {
     return circle.setDate(date || new Date());
 };
 
+function convertToUTC(date) {
+    return new Date(Date.UTC(date.getFullYear(), date.getMonth(), date.getDate(), date.getHours(), date.getMinutes(), date.getSeconds()));
```

Comment:
shorter version: `var utc = date => date - date.getTimezoneOffset() * 60000;`

Reply:
@nickrttn the code uses `.startOf('day')` to it's rather `Date.UTC(date.getFullYear(), date.getMonth(), date.getDate())` right?